Snakemake/envs/shapeit.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - main
+dependencies:
+  - shapeit4 = 4.2.2
```

Comment:
Gabriela, can you remove these files? I think this is now all in workflow/

Reply:
removed all of them